### PR TITLE
CI and the usual gubbins

### PR DESCRIPTION
**Pull request by yaleman**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+(Refers|Fixes) #
+
+## Checklist
+
+- [ ] This pr contains no AI generated code
+- [ ] cargo fmt has been run
+- [ ] cargo clippy has been run
+- [ ] cargo test has been run and passes
+- [ ] book chapter included (if relevant)
```
Comment (Firstyear) [inline .github/pull_request_template.md:9]: Minor, don't need book chap/design doc here I think.

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+# copied from https://github.com/curl/curl/blob/master/.github/stale.yml
+# thanks @bagder
+
+# Number of days of inactivity before an issue becomes stale
```
Comment (Firstyear) [inline .github/stale.yml:5]: I'm not a fan of auto-closing issues, so can this be removed?

Comment (yaleman) [reply .github/stale.yml:5]: Sure!

File: .github/pull_request_template.md (hunk 1 above)
Comment (yaleman) [reply .github/pull_request_template.md:9]: Yep, makes sense!

File: .github/stale.yml (hunk 2 above)
Comment (yaleman) [reply .github/stale.yml]: Funnily enough it doesn't actually do anything without an action that uses it, so I've removed it 😄

**Review by Firstyear — APPROVED** [review]

**Pull request merged**